controllers/question.controller.js

```diff
@@ -1,10 +1,46 @@
 const utility = require('../lib/utility.lib');
-const { createQuestion, destroyQuestion } = require('../lib/question.lib');
+const {
+  createQuestion, destroyQuestion, fetchQuestion, fetchQuestions,
+} = require('../lib/question.lib');
 
 const {
   filterValues, formatValues,
 } = utility;
 
+const getQuestion = async (req, res) => {
+  try {
+    const { id } = req.params;
+
+    const question = await fetchQuestion({ id }, true);
+    console.log(id);
+    if (question) {
+      return res.status(200).send({
+        message: 'Successfully got question',
+        question,
+      });
+    }
+    return res.status(404).send({ message: 'No question with such ID' });
+  } catch (error) {
+    return res.status(500).send({ error: error.message || error });
+  }
+};
+
+const getQuestions = async (req, res) => {
+  try {
+    const questions = await fetchQuestions(true);
+
+    if (questions) {
+      return res.status(200).send({
+        message: 'Successfully got questions',
+        questions,
+      });
+    }
+    return res.status(404).send({ message: 'No question in DB' });
+  } catch (error) {
```

Comment:
Well from the user experience perspective, it is better to return empty list than throw error if questions are empty, let the frontend decides what to do in such cases.